### PR TITLE
feat: allow mac use of serial ports

### PR DESCRIPTION
**Pull request by elyssa-a-adams**
Allows macos users to connect to serial ports via setting security settings

### CODE REVIEW

File: macos/Runner/DebugProfile.entitlements [hunk 1]
```diff
@@ -8,5 +8,9 @@
 	<true/>
 	<key>com.apple.security.network.server</key>
 	<true/>
+	<key>com.apple.security.network.client</key>
+    <true/>
```
Comment (mcm001) [inline macos/Runner/DebugProfile.entitlements:12]: This probably wants to be indented?

File: macos/Runner/Release.entitlements [hunk 2]
```diff
@@ -4,5 +4,9 @@
 <dict>
 	<key>com.apple.security.app-sandbox</key>
 	<true/>
+	<key>com.apple.security.network.client</key>
+    <true/>
+	<key>com.apple.security.device.serial</key>
```
Comment (mcm001) [inline macos/Runner/Release.entitlements:9]: Nit: same

File: macos/Runner/DebugProfile.entitlements (hunk 1 above)
Comment (elyssa-a-adams) [reply macos/Runner/DebugProfile.entitlements:12]: it works on my machine 
image

Comment (elyssa-a-adams) [reply macos/Runner/DebugProfile.entitlements]: fixed it :)

**Review by SamHaggans — APPROVED** [review]

**Pull request merged**